src/server/scripts/Spells/spell_dk.cpp

```diff
@@ -1689,6 +1655,73 @@ class spell_dk_death_grip_initial : public SpellScriptLoader
     }
 };
 
+// 49560 - Death Grip (second part)
+class spell_dk_death_grip : public SpellScriptLoader
```

Comment:
Because now, just for Death Grip, there is 3 spell scripts. For convenience, I thought it would be better to put them one after the other in the order they are called